src/Automation/AxeWindowsAutomationException.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Automation
+{
+    /// <summary>
+    /// The exception type thrown for all unhandled errors in Axe.Windows.Automation.
+    /// If an exception was thrown from code not owned by Axe.Windows.Automation,
+    /// that exception will be wrapped in <see cref="AxeWindowsAutomationException"/>
+    /// In the <see cref="Exception.InnerException"/> property.
+    /// </summary>
+    public class AxeWindowsAutomationException : Exception
```

Comment:
>Exception [](start = 49, length = 9)

If we're exposing this as a public class, we should probably include the full range of ctors and serialization methods that Visual Studio will want to add

Reply:
I don't disagree. But we can do that when visual studio complains about it.